### PR TITLE
SAPI-462-makefile updates

### PR DESCRIPTION
**Pull request by bpathak-ch**
Updates for Mesos and Marathon
* Makefile
* routes
* startup script

### CODE REVIEW

**Review by dlgroves-ch — APPROVED** [review]

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    wget -O "${APP_DIR}/private_env" "${CONFIG_URL}/${ENVIRONMENT}/private_env"
+    wget -O "${APP_DIR}/global_env" "${CONFIG_URL}/${ENVIRONMENT}/global_env"
+    wget -O "${APP_DIR}/app_env" "${CONFIG_URL}/${ENVIRONMENT}/${APP_NAME}/env"
+    source "${APP_DIR}/private_env"
+    source "${APP_DIR}/global_env"
+    source "${APP_DIR}/app_env"
+fi
+
+# Read brokers from environment and split on comma
+IFS=',' read -ra BROKERS <<< "${KAFKA_STREAMING_BROKER_ADDR}"
```
Comment (dlgroves-ch) [inline start.sh]: Please remove this logic; cache-broker-url expects a single-value string (and would be better off being used as an env var).

**Review by dlgroves-ch — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+dist: clean build package
+
+.PHONY: xunit-tests
+xunit-tests: GO111MODULE=off
+xunit-tests:
+	go get github.com/tebeka/go2xunit
+	@set -a; go test -v $(TESTS) -run 'Unit' | go2xunit -output $(xunit_output)
+
+.PHONY: lint
+lint: GO111MODULE=off
```
Comment (bpathak-ch) [inline Makefile:67]: need to remove this line as GO111MODULE is already defined 'on' at line12

Comment (bpathak-ch) [reply Makefile:67]: changes reverted back as golint does not support modules

**Review by dlgroves-ch — CHANGES_REQUESTED** [review]
Still unnecessary logic around setting the cache broker URL in the start script.

**Review by dlgroves-ch — APPROVED** [review]

**Pull request merged**